runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java

```diff
@@ -330,6 +330,8 @@ private boolean isConfigEnabledForProjectileId(int projectileId, boolean notify)
 				return notify ? config.isDrakeNotifyEnabled() : config.isDrakeEnabled();
 			case CERB_FIRE:
 				return notify ? config.isCerbFireNotifyEnabled() : config.isCerbFireEnabled();
+			case DEMONIC_GORILLA_BOULDER:
+				return config.isDemonicGorillaEnabled();
```

Comment:
Coming from the initial comment, here you'd want to follow the same guidelines and return the _same_ value

ie `notify ? config.isDemonicGorillaNotifyEnabled() : config.isDemonicGorillaEnabled();`

Reply:
> Coming from the initial comment, here you'd want to follow the same guidelines and return the _same_ value
> 
> ie `notify ? config.isDemonicGorillaNotifyEnabled() : config.isDemonicGorillaEnabled();`